packages/core/src/State.ts

```diff
@@ -342,3 +347,30 @@ export function getPersistedState<
     TResolvedTypesMeta
   >;
 }
+
+function persistContext(contextPart: Record<string, unknown>) {
+  let copy: typeof contextPart | undefined;
+  for (const key in contextPart) {
+    const value = contextPart[key];
+    if (value && typeof value === 'object') {
+      if ('sessionId' in value && 'send' in value && 'ref' in value) {
+        copy ??= Array.isArray(contextPart)
+          ? (contextPart.slice() as typeof contextPart)
+          : { ...contextPart };
+        copy[key] = {
+          xstate$$type: $$ACTOR_TYPE,
```

Comment:
We need to standardize these kinds of keys 😂

Reply:
i hope we don't have to add any more special values but with this in place, a key like `xstate$$type` is already pretty unique and its values can be extended if any need arises